File: TestFolder/JavaTesting/Testing.java
package JavaTesting;
import java.util.Scanner;

public class Testing {
    static Scanner input = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("Hello there.\nWhat is your name?");
        String name = input.nextLine();
        System.out.println("Well then, hello " + name + ".\nHow old are you?");
        int age = input.nextInt();
        System.out.println("So you are " + age + " years old then.\nDo you want to know how old you will be in ten years? Yes/No");
        yesOrNo(age);
        
    }
    public static int ageInTenYears(int age) {
        return age + 10;
    }
    
    public static void yesOrNo(int age) {
        String yesOrNo = input.next();
        if (yesOrNo.equalsIgnoreCase("Yes")) {
            System.out.println("In ten years you will be " + ageInTenYears(age) + " years old.");
        } else if (yesOrNo.equalsIgnoreCase("no")) {
            System.out.println("Okay then, have a nice day scaredy cat.");
        } else {
            System.out.println("Please type yes or no");
            yesOrNo(age);
        }
    }
}

File: TestFolder/C++/Testing.cpp
#include <iostream>
#include <string>

using namespace std;

int main(int argc, char *arg)
{
    string userName;
    cout << "Hello there, what is your first name? ";
    cin >> userName;
    cout << endl << userName << " is a nice name\nHow old are you? ";
    int userAge;
    cin >> userAge;
    cout << endl << "So you are " << userAge << " years old\nWould you like to know how old you will be in 10 years? ";
    string userAnswer;
    cin >> userAnswer; 

    return 0;
}